x-pack/plugins/alerts/server/saved_objects/index.ts

```diff
@@ -9,6 +9,20 @@ import mappings from './mappings.json';
 import { getMigrations } from './migrations';
 import { EncryptedSavedObjectsPluginSetup } from '../../../encrypted_saved_objects/server';
 
+export const AlertAttributesExcludedFromAAD = [
+  'scheduledTaskId',
+  'muteAll',
+  'mutedInstanceIds',
+  'updatedBy',
+];
```

Comment:
It would be worth linking this to the values we pass into ESO here:

https://github.com/elastic/kibana/blob/e985f1a2725e248c5160e4fc02e946434c57ab85/x-pack/plugins/alerts/server/saved_objects/index.ts#L28

That way they can't go out of sync

Reply:
I already did - easier to see if you look at the complete file instead of the diff; in particular, line 42 (of course!) - or maybe I'm misunderstanding.  

At one point I had all this stuff separated across the two files, decided it would be better to have it all in one place, hopefully the type and attr names array will be easy to keep in sync.

https://github.com/elastic/kibana/blob/384d0b69e1bcaacb01b37741aaa058f40df73c30/x-pack/plugins/alerts/server/saved_objects/index.ts

Reply:
oh, weird, not sure how I missed that 😳

Reply:
TIL something like this should work:

```ts
export const AlertAttributesExcludedFromAAD = [
  'scheduledTaskId',
  'muteAll',
  'mutedInstanceIds',
  'updatedBy',
];
export type AlertAttributesExcludedFromAADType = typeof AlertAttributesExcludedFromAAD[number];
```